### DISCUSSION TITLE
Bi-Weekly Meeting

### DISCUSSION THREAD

**Discussion by frankkilcommins** [OAI/Arazzo-Specification#5] [Announcements]
See https://www.openapis.org/get-involved/about for meeting schedule.

You can also keep an eye on https://github.com/OAI/Arazzo-Specification/issues?q=label%3Ameetings for specific meeting agendas etc.

**Comment by mjang** [comment]
Since this is bi-weekly, is the next meeting on July 2 or July 9 (2025)?

**Reply by frankkilcommins** [reply]
Hi @mjang, 

The next meeting is planned for <strike>July 11th</strike> July 9th.

See https://www.openapis.org/get-involved/about for calendar.

**Reply by mjang** [reply]
Thanks @frankkilcommins . So to confirm, Arazzo meetings are part of the Travel SIG?

![Screenshot 2025-06-30 at 11 12 31 AM](https://github.com/user-attachments/assets/509a0f58-ed82-4468-b50c-3cf9eeb91cb7)

**Reply by frankkilcommins** [reply]
@mjang apologies I obviously had the June calendar in front of me when I responded. Next scheduled Arazzo meeting is on July 9th under the heading of `OpenAPI Workflow SIG`

**Reply by mjang** [reply]
Ah thanks, I get it -- Arazzo is now under "OpenAPI Workflow SIG". Hope to see you there!
